assets/files.wxs

```diff
@@ -1340,9 +1340,6 @@
         <Component Id="cmpB2DC1821DBE648078AEA75AC0AB62408" Guid="{4242bbb5-81dd-481d-a46a-6397d5bce4a5}">
```

Comment:
@SteveL-MSFT, your last commit had 4 failures in `PowerShell-CI-windows`
JEA session Transcript script test.Configuration name should be in the transcript header
```powershell
Cannot bind argument to parameter 'Path' because it is null.
at <ScriptBlock>, D:\a\1\s\test\powershell\engine\Remoting\RemoteSession.Basic.Tests.ps1: line 70
70:             $header = Get-Content $headerFile | Out-String

```
Remote runspace pool should expose commands in endpoint configuration.Error occurred in Describe block
```powershell
Exception calling "Open" with "0" argument(s): "Processing data from remote server localhost failed with the following error message: Could not load file or assembly 'System.Windows.Forms, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089'. The system cannot find the file specified. For more information, see the about_Remote_Troubleshooting Help topic."
At D:\a\1\s\test\tools\Modules\HelpersRemoting\HelpersRemoting.psm1:106 char:5
```
Implicit remoting on restricted ISS.Error occurred in Describe block
```powershell
Expected a value, but got $null or empty.
At D:\a\1\s\src\powershell-win-core\bin\Release\netcoreapp3.0\win7-x64\publish\Modules\Pester\4.8.0\Functions\Assertions\Should.ps1:186 char:9
```
Import-PSSession on Restricted Session.Verifies that Import-PSSession works on a restricted session
```powershell
Cannot validate argument on parameter 'Session'. The argument is null. Provide a valid value for the argument, and then try running the command again.
at <ScriptBlock>, D:\a\1\s\test\powershell\Modules\Microsoft.PowerShell.Utility\Implicit.Remoting.Tests.ps1: line 2067
2067:                 $module = Import-PSSession -Session $session -AllowClobber -ErrorVariable $errorVariable -CommandName Get-Help

```